src/reducers/index.js

```diff
@@ -38,9 +21,10 @@ const counterReducer = (state = initialState.counter, action) => {
   }
 };
 
+const falcorReducerWrapper = (state = initialState.falcor, action) => (falcorReducer(state, action)); // eslint-disable-line max-len
```

Comment:
For setting the initial falcor data you can just change the initialization of your falcor object to something like:

```js
new Falcor({
  cache: initialData,
});
```

Then when `redux-falcor` pulls from faclor the first time the promise will resolve immediately. This will still have the issue where the data won't be there on first render though.